react/index.tsx

```diff
@@ -6,54 +6,78 @@ const CSS_HANDLES = ['container'] as const
 
 enum Positions {
   BOTTOM = 'bottom',
+  TOP = 'top',
 }
 
 interface Props {
   position?: Positions
+  verticalSpacing?: string
```

Comment:
Since this value is used later on as a value in `pixels`, maybe this could be a `number` type?
Then in the `blocks.json` file the user would pass a number to this prop.

Reply:
Done.